css/desktop.css

```diff
@@ -0,0 +1,130 @@
+@media (min-width: 786px) {
+  .navbar .hamburger {
+    display: none;
+  }
+
+  .navbar .links {
+    display: initial;
+  }
+
+  .links .close-btn {
+    display: none;
+  }
+
+  .links .nav-links {
+    display: flex;
+    align-items: center;
+    gap: 32px;
+  }
+
+  .nav-links a {
+    font-weight: 500;
+    text-transform: capitalize;
+    font-family: "Poppins", sans-serif;
+    color: var(--text-purple);
+  }
+
+  .main {
+    width: 100%;
+    background: var(--White);
+    height: auto;
+    padding: 50px 0 80px;
+  }
+
+  .heading-dark {
+    font-size: 2.5rem;
+  }
+
+  .bold-dark,
+  .bold-light {
+    font-size: 1.125rem;
+  }
+
+  /* .main .introduction {
+        background: url(../images/Header-bg.png);
+        background-repeat: no-repeat;
+        background-size: cover;
+    } */
+  .main .introduction {
+    background: url('../images/header-desktop-background.png'), var(--White);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 90vh;
+    width: 95%;
+    margin: 30px auto;
+    background-size: 100% 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
```

Comment:
You don't have to rewrite your `css` for each class in the media query, just write the ones that need to be different.